ifaddrs.go

```diff
@@ -236,38 +236,18 @@ func IfAttrs(selectorName string, ifAddrs IfAddrs) (string, error) {
 	return attrVal, err
 }
 
-// GetAllInterfaces iterates over all available network interfaces and finds all
-// available IP addresses on each interface and converts them to
-// sockaddr.IPAddrs, and returning the result as an array of IfAddr.
-func GetAllInterfaces() (IfAddrs, error) {
```

Comment:
I think this refactor is unnecessary -  if you could revert the change to `GetAllInterfaces()` and stub out `net.Interfaces()` as described in another comment, we should be able to test this function easily